charts/jaeger/templates/collector-svc.yaml

```diff
@@ -46,5 +46,8 @@ spec:
     {{- include "jaeger.selectorLabels" . | nindent 4 }}
     app.kubernetes.io/component: collector
   type: {{ .Values.collector.service.type }}
+{{- if and (eq .Values.collector.service.type "LoadBalancer") .Values.collector.service.loadBalancerIP }}
+    LoadBalancerIP: {{ .Values.collector.service.type.loadBalancerIP }}
```

Comment:
```suggestion
    loadBalancerIP: {{ .Values.collector.service.loadBalancerIP }}
```

Reply:
loadBalancer, should start with lowercase here as per suggestion ⬆️

Reply:
Sure, Ill make the change as well.